USAGE.md

```diff
@@ -18,10 +18,15 @@ curl -X 'POST' \
   "aspects": [
```

Comment:
I noticed, that the Authorization header is also outdated. Please adjust L16 to:
`-H 'X-API-KEY: <api_key>' \`

Reply:
Please do so - if this is necessary. Thank you.